tensorflow_probability/python/experimental/stats/sample_stats.py

```diff
@@ -389,3 +392,152 @@ def finalize(self, state):
       mean: An estimate of the mean.
     """
     return state.mean
+
+
+RunningCentralMomentsState = collections.namedtuple(
+    'RunningCentralMomentsState',
+    'num_samples, mean, sum_exponentiated_residuals')
+
+
+class RunningCentralMoments(object):
+  """Holds metadata for and computes running central moments.
+
+  `RunningCentralMoments` will compute an arbitrary central moment in
+  streaming fashion following the formula proposed by Philippe Pebay
+  (2008) [1]. For reference, the formula we refer to is the incremental
+  version of arbitrary moments (equation 2.9). Since the algorithm computes
+  moments as a function of lower ones, even if not requested, all lower
```

Comment:
Maybe we should embrace this fact on the API level, and alter the names of things like `moment` to `max_moment` and just accept that we're returning a bunch of moments. The class name is already in plural form rather than singular, which is already a step in this direction.

Reply:
Yup, that's definitely an option. I'll go ahead and adopt that then